pages/careers-employment/index.md

```diff
@@ -211,9 +211,9 @@ We can support you in all stages of your job search—from returning to work wit
 <div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
     <h4 class="usa-alert-heading">How do I get help if I'm homeless or at risk of becoming homeless?</h4>
-    <div id="crisis-line-details" class="form-expanding-group borderless-alert additional-info-container" data-event= "nav-crisis-homelessness-expander">
+    <div data-analytics="nav-crisis-homelessness-expander" class="form-expanding-group borderless-alert additional-info-container">
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/9961 removes the need for an ID, and changes `data-event` to `data-analytics`.